src/editor.jsx

```diff
@@ -311,8 +315,22 @@ var imageUrlsFromContent = function(content) {
 
 var Editor = React.createClass({
     propTypes: {
-        imageUploader: React.PropTypes.func,
         apiOptions: ApiOptions.propTypes,
+        className: React.PropTypes.string,
+        content: React.PropTypes.string,
+        disabled: React.PropTypes.bool,
+        // We don't use EnabledFeatures.propTypes here because it requires the
```

Comment:
This is/was a bit hard to fix because each key individually on enabledFeatures is .required, but overall as a whole enabledfeatures should be optional. Maybe it's possible to denote this correctly now! (It may also have been before and I assumed otherwise? Or maybe the defaulting logic for enabledFeatures is complicated [seems reasonably likely but i can't remember?])